packages/flutter_image/example/lib/main.dart

```diff
@@ -0,0 +1,67 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+import 'package:flutter_image/flutter_image.dart';
+
+void main() => runApp(const MyApp());
+
+/// The main app
+class MyApp extends StatelessWidget {
+  /// Contructs main app
+  const MyApp({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    return const MaterialApp(
+      title: 'flutter_image example app',
+      home: HomeScreen(imageUrl: 'https://picsum.photos/250?image=9'),
```

Comment:
Our examples should not use random URLs we don't control. This should point to a URL within the github repository; see https://github.com/flutter/packages/blob/441895b7b1408b5693032e155c34a56244b85a84/packages/video_player/video_player_web/example/integration_test/utils.dart#L16-L23 for an example of what that looks like.

Since we'll need the resource to already exist, you can use a Flutter icon png from a different package that's already checked in, and add a TODO to change it to this package after it lands.